ozone-site.xml

```diff
@@ -15,6 +15,7 @@
    limitations under the License.
 -->
 <configuration>
+<property><name>ozone.om.ratis.enable</name><value>false</value></property>
```

Comment:
How about keeping OM Ratis enabled, but configuring its directories not to conflict with SCM's dir?

```suggestion
<property><name>ozone.om.ratis.storage.dir</name><value>/data/metadata/om/ratis/storage</value></property>
<property><name>ozone.om.ratis.snapshot.dir</name><value>/data/metadata/om/ratis/snapshot</value></property>
```